build.sbt

```diff
@@ -37,7 +37,7 @@ def buildLevelSettings: Seq[Setting[_]] =
       resolvers += Resolver.mavenLocal,
       scalafmtOnCompile := true,
       scalafmtOnCompile in Sbt := false,
-      scalafmtVersion := "1.3.0",
+      scalafmtVersion := "1.4.0",
```

Comment:
Scalafmt 1.5 is now out. Maybe we should bump it up now?

Reply:
I chose not to because of https://twitter.com/olafurpg/status/988665279902339073.